lib/tapioca/runtime/trackers/mixin.rb

```diff
@@ -7,7 +7,8 @@ module Trackers
       module Mixin
         extend T::Sig
 
-        @mixin_map = {}.compare_by_identity
+        @constants_to_mixin_locations = {}.compare_by_identity
```

Comment:
I changed the name of this variable from `mixin_map` to `constants_to_mixin_location`. I had a really hard time remembering what it was supposed to do and I think making it clear what it is mapping between would be helpful. Open to thoughts and name suggestions!